### DISCUSSION TITLE
meaning of server-hello variable in the skeleton code

### DISCUSSION THREAD

**Discussion by Clov4r9** [ANLAB-KAIST/KENSv3#108] [Project Q&A 1. Application Layer (Socket)] [answered]
At Simple Echo Server 3-a,
**"if request := hello\n,response := server-hello($)server-hello variable in the skeleton code))\n"**
Here, what does "server-hello variable in the skeleton code" mean? Where can I find the variable? (I can't find it in EchoAssignment.cpp)
Thanks.

**Comment by ghost** [comment, marked as answer]
It is third argument of serverMain func.
```
int EchoAssignment::serverMain(const char *bind_ip, int port, const char *server_hello)  {
...
...
}
```
